src/config.ts

```diff
@@ -21,20 +21,26 @@ try {
 
 export const yamlConfig = _.merge(defaultConfig, customConfig)
 
-export class TransactionLimits implements ITransactionLimits {
-  readonly config
-  readonly level
-
-  constructor({ level }) {
-    this.config = yamlConfig.limits
-    this.level = level
+export const generateLimitConstants = (
```

Comment:
the same as generateFeeConstants

Reply:
Changed to `getGenericLimits` instead to match up with `getTransactionLimits` and `getUserLimits`